openlibrary/core/lending.py

```diff
@@ -230,6 +230,7 @@ def get_groundtruth_availability(ocaid, s3_keys=None):
     including 1-hour borrows"""
     params = '?action=availability&identifier=' + ocaid
     url = S3_LOAN_URL % config_bookreader_host
+    # TODO: This does not handle unexpected responses from the availability server.
     r = requests.post(url + params, data=s3_keys)
     data = r.json().get('lending_status')
```

Comment:
```suggestion
```suggestion
   try:
        response = requests.post(url + params, data=s3_keys)
        response.raise_for_status()
    except requests.HTTPError:
        pass  # # TODO: Handle unexpected responses from the availability server.
    data = r.json().get('lending_status')
```
https://requests.readthedocs.io/en/master/api/#requests.Response.raise_for_status